src/server/components/formRunner/helpers.ts

```diff
@@ -44,18 +51,15 @@ export function getNewSessionWebhookData(
 }
 
 export async function generateFormRunnerWebhookData(
-  list: List,
-  listItem: BaseListItemGetObject,
+  list: Pick<List, "type">,
```

Comment:
interesting - what does this do?

Reply:
`Pick` is a builtin generic. Essentially it takes an object-like type, and "picks" the keys you specify.

this function actually only needs a list object that looks something like

```
{ type: "lawyers" } //removing all other properties.
```

the rest of the properties are irrelevant. 

At the moment two different queries are being performed, one to find the list, and one to find the list item. In future we can optimise this. Just one call to `listItem` is required, and get prisma to return the related list. Something like

```.ts
prisma.listItem.findUnique({
  where: {..},
  include: {
    list: {
      select: {
        type: true
      }
    }
  }
})

// returns 

{
 id: 1,
 reference: '..',
 jsonData: {..},
 list: {
   type: "lawyers"
 },
}

```

fwiw, I didn't want to change too much, but listItem already returns `type`. I'm not sure why we need both to begin with